torchao/quantization/quantize_/int4/int4_groupwise_preshuffle_tensor.py

```diff
@@ -137,10 +177,17 @@ def from_float(
         cls,
         w: torch.Tensor,
         block_size: List[int],
+        activation_dtype: str = "bf16",
```

Comment:
why a string?

Reply:
this is a string in the fbgemm op API

Reply:
I think we should be consistent with other torchao APIs here instead of trying to match fbgemm